specification/kubernetesruntime/KubernetesRuntime.Management/bgppeer.tsp

```diff
@@ -0,0 +1,49 @@
+import "./common.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+
+namespace Microsoft.KubernetesRuntime;
+
+@doc("A BgpPeer resource")
+model BgpPeer is ExtensionResource<BgpPeerProperties> {
```

Comment:
What all types can it extend? You should try to document the expected usage scenarios